### PR TITLE
Contributors

### PR DESCRIPTION
**Pull request by flamecoals**
#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

File: web/src/games/tictactoe/index.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import instructions from './instructions.md';
 export const tictactoeGameDef: IGameDef = {
   code: 'tictactoe',
   name: 'Tic-Tac-Toe',
+  contributors: ['flamecoals'],
```
Comment (jasonharrison) [inline web/src/games/tictactoe/index.ts:9]: ? 

[Add Tic Tac Toe Game by jasonharrison · Pull Request #69 · freeboardgames/FreeBoardGames.org](https://github.com/freeboardgames/FreeBoardGames.org/pull/69)

File: web/src/infra/gameInfo/GameContributors.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { IGameDef } from 'gamesShared/definitions/game';
+import React from 'react';
+import { Typography } from '@material-ui/core';
+
+interface GameContributorsProps {
+  game: IGameDef;
+}
+
+
+export class GameContributors extends React.Component<GameContributorsProps, {}> {
```
Comment (jasonharrison) [inline web/src/infra/gameInfo/GameContributors.tsx:10]: It doesn't really make any difference (so you can ignore it for this PR if you don't want to bother), but this seems like a good candidate for a functional component.

```typescript
export default function GameContributors({game}: GameContributorsProps) {
    // ...
    return <p>foo</p>;
}

File: web/src/games/tictactoe/index.ts (hunk 1 above)
Comment (flamecoals) [reply web/src/games/tictactoe/index.ts:9]: Fixed, sorry, github history was missing this for old code and I just  guessed, I was going to double check before sending to review.

File: web/src/infra/gameInfo/GameContributors.tsx (hunk 2 above)
Comment (flamecoals) [reply web/src/infra/gameInfo/GameContributors.tsx:10]: Done.

**Comment by flamecoals** [comment]
@jasonharrison finished the about page, ready for review.

**Review by jasonharrison — APPROVED** [review]

**Pull request merged**